### PR TITLE
resolved conflicts

### PR DESCRIPTION
**Pull request by shalinee-walkover**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Other information**:

### CODE REVIEW

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 1]
```diff
@@ -9559,6 +9558,12 @@ span#HW_badge_cont:nth-child(3) {
 .open>.dropdown-menu {
     display: block;
 }
+.daterangepicker.dropdown-menu{
+    box-shadow: 0 3px 28px 0 rgba(0,0,0,.14);
+    color: #616161;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9563]: use color var

File: apps/web-giddh/src/assets/css/style-2.scss [hunk 2]
```diff
@@ -9559,6 +9558,12 @@ span#HW_badge_cont:nth-child(3) {
...
+    border: 1px solid #ccc;
```
Comment (ravinderwalkover) [inline apps/web-giddh/src/assets/css/style-2.scss:9564]: use color var

**Review by ravinderwalkover — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:9563]: Use color variables

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (prateek-giddh) [inline apps/web-giddh/src/assets/css/style-2.scss:9564]: Use color variables

**Review by prateek-giddh — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 1 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Done

File: apps/web-giddh/src/assets/css/style-2.scss (hunk 2 above)
Comment (shalinee-walkover) [reply apps/web-giddh/src/assets/css/style-2.scss]: Done

**Review by prateek-giddh — APPROVED** [review]

**Pull request merged**